charts/internal/shoot-system-components/charts/csi-driver-node/values.yaml

```diff
@@ -37,3 +37,19 @@ resources:
       memory: 300Mi
 
 pspDisabled: false
+securityContext:
+  # The node pod must be run as root to bind to the registration/driver sockets
+  runAsNonRoot: false
+  runAsUser: 0
+  runAsGroup: 0
+  fsGroup: 0
+containerSecurityContext:
+  node:
+    readOnlyRootFilesystem: true
```

Comment:
Can some of these changes be reflected to the PodSecurityPolicy?

For example now we have `readOnlyRootFilesystem=true` for all containers. Can the corresponding field in the PSP be set to `true`? 

https://github.com/gardener/gardener-extension-provider-aws/blob/9a70cad0e269a8e3d307e86d083d419abc325673/charts/internal/shoot-system-components/charts/csi-driver-node/templates/podsecuritypolicy.yaml#L29